packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1264,7 +1272,7 @@ class _TabBarViewState extends State<TabBarView> {
         dragStartBehavior: widget.dragStartBehavior,
         controller: _pageController,
         physics: widget.physics == null ? _kTabBarViewPhysics : _kTabBarViewPhysics.applyTo(widget.physics),
-        children: _children,
```

Comment:
since you don't actually use _children any more as far as i can tell, it'd probably make more sense to rename _childrenWithKey to _children.

Reply:
We will still need the reference to _children. do you have any suggestion to the new name of _children?